src/lib/y2storage/boot_requirements_strategies/legacy.rb

```diff
@@ -177,8 +183,7 @@ def errors_on_msdos
       def unknown_boot_partition_table_error
         # TRANSLATORS: error message
         error_message = _(
-          "Boot requirements cannot be determined because " \
-          "boot disk has not partition table"
+          "Boot disk is without partition table"
```

Comment:
I do not like too formal "Boot requirements cannot be determined" it sounds like our error that we cannot determine it. I think we should make it more user oriented. How you will see
`Boot disk is without partition table and we does not support booting from it`

BTW that reminds me that it would be nice to also add to end of message how to fix it. E.g. `You can fix it by creating partition table on disk`.